### PR TITLE
Add Chorus One as a staking provider

### PR DESCRIPTION
**Pull request by liminal-state**
Adding Chorus One as a whitelabel HNT staking provider

### CODE REVIEW

File: docs/mine-hnt/validators/faqs-resources-providers.mdx [hunk 1]
```diff
@@ -114,3 +114,4 @@ Links to these Staking Providers are provided as is. Helium, Inc. is not directl
 * [Vanager](https://vanager.io)
 * [StakeJoy](https://stakejoy.com)
 * [NodeValidators](https://nodevalidators.com)
+* [Chorus One](https:chorus.one/whitelabel-one)
```
Comment (jthiller) [inline docs/mine-hnt/validators/faqs-resources-providers.mdx]: This URL appears to be broken.
```suggestion
* [Chorus One](https://chorus.one/products/whitelabel-staking)
```

**Comment by liminal-state** [comment]
Thanks for correcting https://chorus.one/whitelabel-staking is correct!

**Comment by jthiller** [comment]
It may be advised to publish the page before submitting the PR. 
image

**Comment by liminal-state** [comment]
It's actually https://chorus.one/products/whitelabel-staking

Sorry for the confusion...

**Comment by liminal-state** [comment]
I just corrected with the wrong URL again, sorry about that (see second
cmt, it should be https://chorus.one/products/whitelabel-staking).

**Review by abhay — APPROVED** [review]

**Pull request merged**